grails-app/assets/javascripts/streama/translations/DE_de.js

```diff
@@ -3,7 +3,137 @@
+			LANGUAGE: 'Sprache',
+			PAUSE_ON_CLICK: 'Pausiere Video bei Klick',
+			FAVORITE_GENRES: 'Lieblings-Genres',
+			SAVE: 'Profil speichern',
+			OLD_PASS: 'Altes Passwort',
+			NEW_PASS: 'Neues Passwort',
+			NEW_PASS_PLACEHOLDER: 'Neues Passwort  (min. 6 Zeichen)',
+			REPEAT_PASS: 'Passwort wiederholen',
+			SAVE_PASS: 'Neues Passwort setzen'
+		},
+
+		SORT_OPTIONS: {
+			AZ: 'A-Z',
+			ZA: 'Z-A',
+			NEWEST_ADDED: 'Zuletzt hinzugefügt',
+			OLDEST_ADDED: 'Zuerst hinzugefügt',
+			NEWEST_RELEASED: 'Neuste Veröffentlichung',
+			OLDEST_RELEASED: 'Älteste Veröffentlichung',
+			NEWEST_AIRED: 'Zuletzt angeschaut',
+			OLDEST_AIRED: 'Zuerst angeschaut'
+		},
+
+		FAQ: {
+			UPLOAD_VIDEO: {
+				TITLE: 'Wie kann ich ein Video hochladen?',
+				TEXT: "Du kannst Videos hochladen indem du auf Inhalte verwalten klickst. Wähle aus, ob du eine Serie, einen Film oder ein Video hochladen möchtest. Klicke auf die relevante Option" +
+				"auf der linken Seite. Dann klicke auf den Button rechts. Danach öffent sich die Suchbar und gib den Titel des Film ein. Dies kannst du auch manuell machen, falls keine Vorschläge kommen." + 
+				"Im Anschluss kannst du das Video mit einer Datei verknüpfen oder eine Datei hoch laden."
+
+			},
+			DELETE_VIDEO: {
+				TITLE: 'Wie kann ich ein Video löschen?',
+				TEXT: "Du kannst Videos in der Video Informationsseite löschen, wenn du Inhalte verwalten anklickst und dann auf den roten Mülleimer. Video editieren anklicken und dann
+				Video löschen" +
+				" an zu klicken ist ein anderer Weg ein Video zu löschen. Du kannst auch den Filemanager benutzen, der sich im Inhalte Verwalten Menü befindet." +
+				" Auch hier den roten Mülleimer verwenden."
+			},
+			VIDEO_FORMATS: {
+				TITLE: 'Welche Video Formate werden unterstützt?',
+				TEXT: "Streama unterstützt im Moment nur Formate für den HTML5 player. Du kannst testen ob deine Videodatei HTML5 kompatiebel ist, indem du diese in ein/en leeren " +
+				" Browsertab/fenster ziehst und fallen lässt."
+			},
+			SUBTITLES: {
+				TITLE: 'Wie kann ich Untertitel zu Videos hinzufügen?',
+				TEXT: "Du kannst Untetertitel in der Videos Infromationsseite unter Inhalte verwalten. Du kannst dort Untetitel durch Drag and Drop hinzufügen." +
+				"Die Untertitel müssen nicht mehr konvertiert werden."
+			},
+			INVITE_USERS: {
+				TITLE: 'Wie kann ich Freunde einladen um meine gehosteten Videos an zu schauen?',
+				TEXT:"Du kannst deine Videos auf Streama teilen, indem du deine Frende einlädst. Gehe zum Benutzer Menu und klicke auf den Invite Button. Fülle das Formuular aus" +
+				" und wähle die Rollen. Benutzer mit der Rolle Admin können Benutzer und Einstellungen ändern. Benutzer mit der Rolle role Content Manager können Inhalte verwalten." +
+				" Deine Freunde werden über Email benachrichtigt, das du Sie eingeladen hast. 
+				Du kannst auch Video-Sessions teilen indem du auf den Share Button des Videoplaers drückst und die Session URL teilst."
```

Comment:
You could change "Video-Sessions" to "Videositzungen" and "Session" to "Sitzung" for consistency. Also "Videoplayer" is misspelled.